lib/burp/issue.rb

```diff
@@ -106,19 +107,22 @@ def cleanup_html(source)
       result.gsub!(/<h2>(.*?)<\/h2>/, '*\1*')
       result.gsub!(/<i>(.*?)<\/i>/, '\1')
       result.gsub!(/<p>(.*?)<\/p>/, '\1')
+      result.gsub!(/<p>/, '')
```

Comment:
why is this needed? And shouldn't it be covered by line #109 ?

Reply:
@etdsoft there are instances like this where the `<p>` tags are on a separate line and aren't caught by Line #109: https://github.com/dradis/dradis-burp/blob/21eaec6b5310782329303af31353ff7b118ab599/templates/issue.sample#L11. Is there a better way to deal with these that I didn't think of?

Reply:
what about using a multi-line shallow regexp?

`/<p>(.*?)<\/p>/m`

You can play with test data and regexp's here:

http://rubular.com/